### PR TITLE
Added API Call to retrieve directors

### LINKED ISSUE #86: Server: Retrieve the history of board of directors
Retrieve ID, Name, Role, Date of recruitment and Date of retirement.

### PR DESCRIPTION
**Pull request by KevinGus**
Added api route in membersRoutes. 
Added the functionality of retrieveDirectors in membersController.
Closes #86

### CODE REVIEW

File: server/src/members/membersController.ts [hunk 1]
```diff
@@ -145,6 +145,26 @@ export const createMember = (req: Request, res: Response): void => {
...
 
+/**
+ * Retrieves the history of directors
+ */
+export const retrieveDirectors = (req: Request, res: Response): void => {
+  database.query(
+    `SELECT mitgliedID, vorname, nachname, geschlecht, kuerzel, bezeichnung_maennlich,bezeichnung_weiblich, von, bis 
+    FROM mitglied, mitglied_has_evposten, evposten
+    WHERE mitgliedID = mitglied_mitgliedID AND evpostenID = evposten_evpostenID `, [])
+  .then((result: []) => {
```
Comment (LennartRukower) [inline server/src/members/membersController.ts:156]: Please add a query result type.
See line 66 for an example: ...(result: membersTypes.GetMembersQueryResult)...

**Review by LennartRukower — CHANGES_REQUESTED** [review]
Good pr!

File: server/src/members/membersController.ts [hunk 2]
```diff
@@ -145,6 +145,26 @@ export const createMember = (req: Request, res: Response): void => {
   });
 };
 
+/**
+ * Retrieves the history of directors
+ */
+export const retrieveDirectors = (req: Request, res: Response): void => {
+  database.query(
+    `SELECT mitgliedID, vorname, nachname, geschlecht, kuerzel, bezeichnung_maennlich,bezeichnung_weiblich, von, bis
```
Comment (LennartRukower) [inline server/src/members/membersController.ts:153]: Please remove trailing whitespace.

**Review by LennartRukower — CHANGES_REQUESTED** [review]

**Review by christophtrybek — APPROVED** [review]
LGTM

**Pull request merged**